### ISSUE TITLE
Clean up "successful uploaded" section

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#2339]
- create a job that runs daily(?)
- cleans up "successful uploads" older than 7 days for all accounts

as discussed in https://github.com/nextcloud/android/issues/2324

**Comment by mario** [comment]
Daily is absolutely fine.

**Comment by xXSTrikeXx** [comment]
Then I cannot see anymore how many photos I('ll) take this year :flushed:, is it correct?

**Comment by mario** [comment]
Yes. Use different means of counting them :)

**Comment by xXSTrikeXx** [comment]
@mario which means? :open_mouth:

**Comment by AndyScherzinger** [comment]
:+1: fine by me. 

> hen I cannot see anymore how many photos I('ll) take this year

What is the benefit of this? While beware that you don't know either today since any kind of auto upload will be in your list (e.g. when having custom folders or videos) or whatsApp or Instagram since for WhatApp it would be the pictures you received also ;)

**Comment by xXSTrikeXx** [comment]
This is not my case since (WA was bought by FB) and I reduce smartphone consumption to calls. Its most clear and no text messenger can replace it. So I only take photos with my phone. :) And there is no other implementation to count files in the app. This could be an enhancement to show the counting on instant autoupload folders (e.g. 2018). Im just brainstorming.

**Comment by AndyScherzinger** [comment]
@xXSTrikeXx agreed, while looking at it from a "all users" perspective the argumnet would have to be applied to any user of the app and other will most definitely have other uploads as soon as they activate the auto upload for any kind of media folder filled by an app that makes use of it ;)
